src/Backend/Modules/ContentBlocks/Domain/ContentBlock/ContentBlockRepository.php

```diff
@@ -38,20 +38,43 @@ public function getNextIdForLanguage(Locale $locale): int
             ->getSingleScalarResult() + 1;
     }
 
-    public function findOneByIdAndLocale($id, Locale $locale): ?ContentBlock
+    public function findOneByIdAndLocale(int $id = null, Locale $locale): ?ContentBlock
```

Comment:
wrong method signature, if the int needs to be able to receive a null (maybe check the implementation) you need to use `?int` and not `= null` because now you have an optional parameter before a required one